src/metabase/models/serialization.clj

```diff
@@ -387,8 +407,11 @@
         (assoc :serdes/meta (generate-path model-name entity))
         (dissoc pk :updated_at))))
 
-(defmethod extract-one :default [model-name _opts entity]
-  (extract-one-basics model-name entity))
+(defmethod extract-one :default [model-name opts entity]
+  ;; `extract-by-spec` is called here since most of tests use `extract-one` right now
```

Comment:
Well if `extract-one` becomes a simple function (rather than a multimethod) which operates on spec then this comment won't be necessary. For now it just explains why I did not find a better place. :)